metadata-ingestion/src/datahub/ingestion/source/redshift/redshift_schema.py

```diff
@@ -242,11 +243,15 @@ def __init__(self, is_serverless):
     def get_query_result(
         conn: redshift_connector.Connection, query: str
     ) -> redshift_connector.Cursor:
-        cursor: redshift_connector.Cursor = conn.cursor()
-
-        logger.debug(f"Query : {query}")
-        cursor.execute(query)
-        return cursor
+        with PerfTimer() as timer, conn.cursor() as cursor:
+            query_fragment = re.sub(r"\s+", " ", query).strip()[:100] + "..."
```

Comment:
Also this will always add the ... dots even if the query is shorter than 100 chars

Reply:
Also see my comment on the left side.